app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :event
+
+  has_many :answers, dependent: :destroy 
+
+  validates :body, presence: true, allow_blank: false
+end
```

Comment:
Add EOF please

Reply:
I don't understand? there is already the end?

Reply:
After the final `end` there should be a line, please check this example:
<pre>
```ruby
end

```
</pre>
Note that there is an "empty line"

Reply:
ok, i verify it and push today